shared/js/background/declarative-net-request.js

```diff
@@ -327,6 +331,16 @@ export async function refreshUserAllowlistRules (allowlistedDomains) {
     await updateUserAllowlistRule(normalizedAllowlistedDomains)
 }
 
+export async function flushSessionRules () {
+    chrome.declarativeNetRequest.getSessionRules().then(rules => {
+        if(rules.length) {
+            console.log('removing session rules', ruleIds)
```

Comment:
Maybe remove this comment? Or if not, we should at least move this below where we define `ruleIds`.